params/params.yaml

```diff
@@ -0,0 +1,172 @@
+amcl:
+  ros__parameters:
+    laser_max_range: -1.0 # Use value reported in laser scan
```

Comment:
I'm assuming this has already been tested but this looks bizarre, any further details?

Reply:
Yes, see the [AMCL configuration docs](https://navigation.ros.org/configuration/packages/configuring-amcl.html).